runtests.py

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env python3
 from unittest import TestCase
+
+from slack_sdk.socket_mode.request import SocketModeRequest
+
```

Comment:
nit: Can we regroup these? `unittest` below should be grouped with unittest above, then 3rd party stuff in a group, and then util.